tests/test_types.py

```diff
@@ -499,6 +499,8 @@ class MoreStringsModel(BaseModel):
         str_min_length: constr(min_length=5) = ...
         str_curtailed: constr(curtail_length=5) = ...
         str_email: EmailStr = ...
+        str_email_local_case_sensitive: EmailStr = ...
```

Comment:
I don't think we need this, this section of the docs is already too long.

Reply:
ok, removed :zap: